src/System.Private.CoreLib/shared/System/Runtime/InteropServices/SafeBuffer.cs

```diff
@@ -110,13 +110,17 @@ public void Initialize(ulong numBytes)
         [CLSCompliant(false)]
         public void Initialize(uint numElements, uint sizeOfEachElement)
         {
```

Comment:
@jkoritzinsky I have just realize that this can be simplified even further by just calling the other overload, like:
```
public void Initialize(uint numElements, uint sizeOfEachElement)
{
    Initialize((long)numElements * sizeOfEachElement);
}
```